src/collective/liveblog/upgrades/v1002/__init__.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding:utf-8 -*-
+from collective.liveblog.config import PROJECTNAME
+from collective.liveblog.interfaces import ILiveblog
+from plone import api
+from plone.app.upgrade.utils import loadMigrationProfile
+
+import logging
+
+
+logger = logging.getLogger(PROJECTNAME)
+
+
+def remove_workflow(context):
```

Comment:
I can't find anything; ask in SO or in the community forums; at the end what we need is that all liveblogs in `private` state remain the same and all liveblogs in `active` or `inactive` states must be `published`.